central/cloudsources/datastore/datastore.go

```diff
@@ -13,8 +13,6 @@ import (
 )
 
 // DataStore is the entry point for modifying cloud sources data.
-//
-//go:generate mockgen-wrapper
```

Comment:
Why don't we need it anymore?

Reply:
We originally had tests within the service layer that were using the datastore mock, I just noticed that since we don't have  those tests anymore we don't need the generate statement anymore.